src/common/pick_address.cc

```diff
@@ -237,8 +237,16 @@ static int fill_in_one_address(
   const struct sockaddr *found = find_ip_in_subnet_list(cct, ifa, ipv, networks,
 							interfaces, numa_node);
   if (!found) {
-    lderr(cct) << "unable to find any IP address in networks '" << networks
-	       << "' interfaces '" << interfaces << "'" << dendl;
+    std::string ip_type = "";
+    if ((ipv & CEPH_PICK_ADDRESS_IPV4) && (ipv & CEPH_PICK_ADDRESS_IPV6)) {
+      ip_type = "IPv4 or IPv6";
```

Comment:
But Ceph can't run dual stacked, so why would we even add this option?

IPv4 and IPv6 are mutually exclusive with Ceph

Reply:
with the infrastructure for the v2 protocol we have most of the pieces we need to run dual stack.  i expect it's a modest effort to close the gap.  this is one of the pieces